### ISSUE TITLE
`Dataplanes` page of `Kuma GUI` fails with an error if a `Dataplane` has 8 tags or more

### ISSUE THREAD

**Issue by yskopets** [Kong/kuma#618]
### Summary

* `Dataplanes` page of `Kuma GUI` fails with an error in some cases

### Steps To Reproduce

1. Create a `Dataplane` with 8 tags (or more), e.g.
   ```shell
   echo "
   type: Dataplane
   name: backend-01
   mesh: default
   networking:
     address: 127.0.0.1
     inbound:
     - port: 80
       servicePort: 8080
       tags:
         service: backend
         tag02: value02
         tag03: value03
         tag04: value04
         tag05: value05
         tag06: value06
         tag07: value07
         tag08: value08
   " | kumactl apply -f -
   ```
2. Go to `Dataplanes` page on `Kuma GUI` - http://localhost:5683/#/default/dataplanes
3. The page doesn't show any `Dataplanes` and there is a JavaScript error logged in the Console

![image](https://user-images.githubusercontent.com/19395458/75693591-71659d00-5ca7-11ea-842c-f9a1232882d5.png)

### Additional Details & Logs

- Version; `0.4.0`
- Platform and Operating System: `MacOS`

**Comment by bloqhead** [comment]
@yskopets thank you for submitting this! I've fixed the error and was successfully able to have 8+ tags. I've created a PR [here](https://github.com/Kong/kuma-gui/pull/13) and provided a description for what I changed and why the error occurred.

**Comment by nickolaev** [comment]
This is fixed with #685. Closing.
